tests/test_buttons.py

```diff
@@ -9,52 +53,93 @@ def test_tactile_buttons_initialization(mocker, m5stickv):
     assert not buttons_control.page_prev_event_flag
 
 
-def test_button_enter_event(mocker, m5stickv):
-    from krux.buttons import ButtonEnter, buttons_control
+def test_generic_button_default_values(mocker, m5stickv, mock_tactile_buttons):
+    from krux.buttons import Button, RELEASED
 
-    # Simulate button enter pin
-    mocked_pin = 10
-    pin = mocked_pin
-    button_enter = ButtonEnter(pin)
-    # Overwrite the pin's MagicMock with the mocked pin number
-    buttons_control.enter = mocked_pin
+    button = Button()
 
-    # Simulate button press
-    buttons_control.event_handler(pin)
-    assert button_enter.event()
-    # Assert that the event flag is reset
-    assert not button_enter.event()
+    # Assert button methods return default values
+    assert button.value() == RELEASED
+    assert not button.event()
+    mock_tactile_buttons.event_handler.assert_not_called()
 
 
-def test_button_page_event(mocker, m5stickv):
-    from krux.buttons import ButtonPage, buttons_control
+def test_no_button_enter(mocker, m5stickv, mock_tactile_buttons):
+    from krux.buttons import ButtonEnter, RELEASED
 
-    # Simulate button page pin
-    mocked_pin = 11
-    pin = mocked_pin
-    button_page = ButtonPage(pin)
-    # Overwrite the pin's MagicMock with the mocked pin number
-    buttons_control.page = mocked_pin
+    # Simulate device has no button enter
+    mocker.patch.object(mock_tactile_buttons, "init_enter", lambda pin: None)
+    mock_tactile_buttons.enter = None
+    button_enter = ButtonEnter(None)
```

Comment:
This button is being created alone, not as part of `mock_tactile_buttons`(which the goal is to mock `buttons_control`, right?). In addition, we don't create a ButtonX instance with `None` as argument.
To test the case where there's no button ENTER (to be coherent with test title), you can use `mock_tactile_buttons.enter = None`, as you did, but `ButtonEnter` is never instantiated. It seems what you are doing here is to test button's default values again.